src/translations/en.json

```diff
@@ -67,5 +67,10 @@
     "profiles.form.modification.updateError": "Error while updating the profile",
 
     "linked.path.display.noLink": "no parameters selected.",
-    "linked.path.display.invalidLink": "invalid parameters link."
+    "linked.path.display.invalidLink": "invalid parameters link.",
+
+    "users.form.delete.title": "Delete a user",
+    "users.form.delete.multipleItemsDialogMessage": "{itemsCount} users will be deleted.",
+    "users.form.delete.itemDialogMessage": "{itemName} will be deleted.",
+    "users.form.delete.multipleItemsDialogTitle": "Delete {itemsCount} users"
```

Comment:
suggestion for the key names:
"users.form.delete.dialog.title"
"users.form.delete.multiple.dialog.message"
"users.form.delete.dialog.message"
"users.form.delete.multiple.dialog.title"

And move up this scope after all other "users.* keys

Reply:
Done.